docs/examples/python/scell_test_postgres.py

```diff
@@ -100,7 +101,7 @@ def get_record(connection, id):
 
 if __name__ == '__main__':
     dsn = ("dbname=scell_token_protect_test user=postgres password=postgres "
-           "host=172.17.0.2")
```

Comment:
now it looks like somebody use it and test and it doesn't contain hardcoded values)